libs/cardano-ledger-binary/cardano-ledger-binary.cabal

```diff
@@ -158,5 +158,4 @@ test-suite tests
                    , vector
                    , vector-map
 
-  ghc-options:      -threaded
-                    -rtsopts
+  ghc-options:      -threaded -rtsopts -with-rtsopts=-N
```

Comment:
`with-rtsopts=N` is for parallelism?  (Sorry, can't find the specs the opts passed to -with-rtsopts` ) .
Any reason for which  it's set for this particular package? ( I see a few more which have it too).  Is it to do with the serialization?

Reply:
Yeah, That's a very good question. What `-N` actually means is use all cores that are available on the machine. It is also possible to specify a concrete number too, eg: `-N4`, which will tell RTS to use  4 capabilities (i.e. OS threads), and if there are at least 4 cores than it will be able to utilize them.

 I don't remember why I added multi-core execution here, might have been working on multiple projects at the same time and mixed em up. That being said, it is possible to parallelize tests and we should probably experiment with it, hspec has this ability out of the box with [`parallel`](https://hackage.haskell.org/package/hspec-2.10.8/docs/Test-Hspec.html#v:parallel). So, I'll leave the `-N` here, since it doesn't hurt and can be useful in the future.